### PR TITLE
Reworking pg upgrade process and adding confirmation prompt

### PR DESCRIPTION
**Pull request by davissp14**
**Notable changes:**

* Adds a confirmation prompt before performing update.
* Ensures any non-pg machines running under the same app are ignored.
* Added check to ensure PG machines do not span major versions. 

Screen Shot 2022-09-29 at 4 17 15 PM

### CODE REVIEW

**Review by dangra — APPROVED** [review]
Looks great! Are you going to add the health check test before merging or is that for a follow up PR?

File: internal/command/postgres/update.go [hunk 1]
```diff
@@ -150,43 +191,34 @@ func runUpdate(ctx context.Context) error {
...
 			pgclient := flypg.New(app.Name, dialer)
 
-			fmt.Fprintf(io.Out, "Performing a failover\n")
+			fmt.Fprintf(io.Out, "Performing failover\n")
 			if err := pgclient.Failover(ctx); err != nil {
 				return fmt.Errorf("failed to trigger failover %w", err)
 			}
 		}
 
 		fmt.Fprintf(io.Out, "Updating leader\n")
```
Comment (dangra) [inline internal/command/postgres/update.go:218]: nitpick: the message is miss leading (pun-intended) in case of a failover  🔍

**Comment by davissp14** [comment]
> Looks great! Are you going to add the health check test before merging or is that for a follow up PR?

That will have to be a follow-up PR as some of it will rely on @rugwirobaker 's on-going work.

File: internal/command/postgres/update.go (hunk 1 above)
Comment (dangra) [reply internal/command/postgres/update.go]: heh, the weird part is that the failover isn't happening for single node postgres. Not a big deal

**Pull request merged**